order/src/main/resources/application.properties

```diff
@@ -14,8 +14,8 @@ yas.services.customer=http://api.yas.local/customer
 
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.url=jdbc:postgresql://localhost:5432/order
-spring.datasource.username=admin
-spring.datasource.password=admin
+spring.datasource.username=postgres
+spring.datasource.password=123456
```

Comment:
Does it work? 
Database postgres exposed from docker has username is admin, and password is admin.